weekly/2023/week-32.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Week 32 in Packit
 date: 2023-08-14
+authors: flachman
```

Comment:
I was kinda contemplating about adding this to the template, but I didn't want to force anyone to use it :D

Reply:
I was also not sure if this makes sense. But at least it's visible there are humans behind Packit..;)